src/openms/include/OpenMS/ANALYSIS/ID/TopPerc.h

```diff
@@ -0,0 +1,200 @@
+         * @param new_peptide_ids PeptideIdentification vector to be appended
+         * @param search_engine search engine to depend on for feature creation
+         *
+         * Appends a vector of PeptideIdentification to another and registers concatenation Percolator features depending on given search engine.
+         */
+        static void concatMULTISEPeptideIds(std::vector<PeptideIdentification>& all_peptide_ids, std::vector<PeptideIdentification>& new_peptide_ids, String search_engine);
+
+        /**
+         * @brief mergeMULTISEPeptideIds
+         * @param all_peptide_ids PeptideIdentification vector to be merged into
+         * @param new_peptide_ids PeptideIdentification vector to merge
+         * @param search_engine search engine to depend on for feature creation
+         *
+         * Merges a vector of PeptideIdentification into another and registers merge Percolator features depending on given search engine.
+         */
+        static void mergeMULTISEPeptideIds(std::vector<PeptideIdentification>& all_peptide_ids, std::vector<PeptideIdentification>& new_peptide_ids, String search_engine);
+
+        /**
+         * @brief mergeMULTISEProteinIds
+         * @param all_protein_ids ProteinIdentification vector to be merged into
+         * @param new_protein_ids ProteinIdentification vector to merge
+         *
+         * Concatenates SearchParameter of multiple search engine runs and merges PeptideEvidences, registers the created Percolator features
+         */
+        static void mergeMULTISEProteinIds(std::vector<ProteinIdentification>& all_protein_ids, std::vector<ProteinIdentification>& new_protein_ids);
+        
+
+        /**
+         * @brief addMSGFFeatures
+         * @param peptide_ids PeptideIdentification vector to create Percolator featrues in
+         * @param feature_set register of added features
+         *
+         * Creates and adds MSGF+ specific Percolator features and registers them in feature_set
+         */
+        static void addMSGFFeatures(std::vector<PeptideIdentification>& peptide_ids, StringList& feature_set);
+
+        /**
+         * @brief addXTANDEMFeatures
+         * @param peptide_ids PeptideIdentification vector to create Percolator featrues in
+         * @param feature_set register of added features
+         *
+         * Creates and adds X!Tandem specific Percolator features and registers them in feature_set
+         */
+        static void addXTANDEMFeatures(std::vector<PeptideIdentification>& peptide_ids, StringList& feature_set);
+
+        /**
+         * @brief addCOMETFeatures
+         * @param peptide_ids PeptideIdentification vector to create Percolator featrues in
+         * @param feature_set register of added features
+         *
+         * Creates and adds Comet specific Percolator features and registers them in feature_set
+         */
+        static void addCOMETFeatures(std::vector<PeptideIdentification>& peptide_ids, StringList& feature_set);
```

Comment:
e.g., here and in the examples above it is not clear what "create and add" means. Do you compute them? Or are these just extracted.